packages/pds/src/api/app/bsky/feed/getRepostedBy.ts

```diff
@@ -30,14 +31,14 @@ export default function (server: Server) {
         builder = builder.where('repost.subjectCid', '=', cid)
       }
 
+      const keyset = new RepostsKeyset()
```

Comment:
think it could be nice to be able to tell what the cursor is composed of inline. would be pretty straightforward if the primary/secondary vals were passed through the constructor 🤔